### PR TITLE
add individual taskqueue

### PR DESCRIPTION
**Pull request by GuohaoOuyang**

### CODE REVIEW

**Comment by nalanart** [comment]
This looks good to me!

File: server/controllers/userController.js [hunk 1]
```diff
@@ -25,24 +25,24 @@ const signUp = async (req, res) => {
     });
 
     if (user) {
+      handleIndividualCompany(user.name);
```
Comment (nalanart) [inline server/controllers/userController.js]: should be changed here as well to user.activeCompany

**Review by nalanart — APPROVED** [review]

File: server/controllers/userController.js (hunk 1 above)
Comment (GuohaoOuyang) [reply server/controllers/userController.js]: Thank you, Alan! Changed it

**Comment by nalanart** [comment]
Can this individual task queue be used when the user adds a new company as well?

**Comment by GuohaoOuyang** [comment]
> Can this individual task queue be used when the user adds a new company as well?

Yes, once the route and endpoints ready, you can access the company by req.body and pass it as params.

**Pull request merged**